templates/patterns/banners/banner.ui_patterns.yml

```diff
@@ -0,0 +1,52 @@
+banner:
+  label: "Banner"
+  description: ""
+  variants:
+    default:
+      label: "Banner: default"
+      description: "Default contains title, description and link with gray background."
+    hero:
+      label: "Banner: hero"
```

Comment:
Why do we need the Hero variant if the type (Page or Hero) is defined by the modifier?